charts/osm/values.yaml

```
@@ -73,30 +73,30 @@ OpenServiceMesh:
   enablePrometheusScraping: true
   # -- Deploy Grafana
   deployGrafana: false
-  # -- Enable Fluentbit sidecar deployment
+  # -- Enable Fluent Bit sidecar deployment
   enableFluentbit: false
   fluentBit:
-    # -- luentBit Sidecar container name
+    # -- FluentBit sidecar container name
```

The smallest of nits:
```suggestion
    # -- Fluent Bit sidecar container name
```